qiskit/providers/models/backendproperties.py

```diff
@@ -122,8 +126,156 @@ def __init__(self, backend_name, backend_version, last_update_date,
+        """
+        return self.gate_property(operation, qubits,
+                                  'gate_length')[0]  # Throw away datetime at index 1
+
+    def gate_property(self,
+                      operation: str,
+                      qubits: Union[int, Iterable[int]] = None,
+                      name: str = None) -> Tuple[Any, datetime.datetime]:
+        """
+        Return the gate properties of the given qubit and property.
+
+        Args:
+            operation (str) : Name of the gate
+            qubits (Union[int, Iterable[int]]): The qubit to find properties for.
+            name (str): Optionally used to specify within the heirarchy which
+                        property to return.
+
+        Return:
+            Gate properties of the given qubit and property, if it was given by the
+            backend, otherwise, return `None`.
+
+        Raises:
+	        PulseError: If `qubits` is `None` and `name` is `not None` or
+            if the property is not found.
+        """
+        result = self._gates
+        try:
+            result = result[operation]
+            if qubits is not None:
+                result = result[_to_tuple(qubits)]
+                if name:
+                    result = result[name]
+            elif name:
+                raise PulseError("Provide qubits to get {n} of '{o}'.".format(n=name, o=operation))
+        except KeyError:
+            raise PulseError("Could not find the desired property.")
+        return result
+
+    def qubit_property(self, qubit: int,
+                       name: str = None) -> Tuple[Any, datetime.datetime]:
+        """
+        Return the qubit properties of the given qubit and name.
+
+        Args:
+            qubit (int): The property to look for.
+            name (str): Optionally used to specify within the heirarchy which property to return.
+
+        Return:
+            Qubit properties of the given qubit and name, if it was given by
+            the backend, otherwise, return `None`.
+
+        Raises:
+	        PulseError: If `qubit` is `None` and `name` is `not None` or
+            if the property is not found.
+        """
+        result = self._qubits
+        try:
+            result = result[qubit]
+            if name is not None:
+                result = result[name]
+        except KeyError:
+            raise PulseError("Please check arguments again. Couldn't find the desired property.")
+        return result
+
+    def t1(self, qubit: int) -> Tuple[Any, datetime.datetime]:
+        """
+        Return the properties of T1 of the given qubit.
+
+        Args:
+            qubit (int): The property to look for.
+
+        Return:
+            Properties of T1 of the given qubit, if it was given by the backend, otherwise,
+            return `None`.
+
+        Raises:
+	        PulseError: If error is True and the property is not found.
+        """
+        return self.qubit_property(qubit, 'T1')
```

Comment:
Return only the value (just like in `gate_error`, we have to index into the result)